Cargo.lock

```diff
@@ -3199,6 +3186,19 @@ dependencies = [
  "serde_json 1.0.81",
 ]
 
+[[package]]
+name = "jsonrpc-core"
+version = "18.0.0"
+source = "git+https://github.com/scs/jsonrpc?branch=no_std_v18#eb7cdf49c0e5d35174f972743a7e3ff460aadedc"
+dependencies = [
+ "futures 0.3.8",
+ "log 0.4.14 (git+https://github.com/mesalock-linux/log-sgx)",
+ "serde 1.0.118 (git+https://github.com/mesalock-linux/serde-sgx)",
+ "serde_derive 1.0.118",
+ "serde_json 1.0.60 (git+https://github.com/mesalock-linux/serde-json-sgx)",
+ "sp-std 4.0.0 (git+https://github.com/paritytech/substrate.git?branch=master)",
```

Comment:
Further down, I said that we might want to swith the `sp-std` to the polkadot branch too, but I think that it's actually ok because `sp-std` does not have any second-level dependencies.

Reply:
I was thinking about that too. But pointing to a polkadot branch would mean we have to update jsonrpc-core library manually to keep up to date. I think pointing to master is less complicated in this case.